docs/deploy.md

```diff
@@ -22,7 +22,7 @@ Make sure that your Kubernetes config (e.g, `~/.kube/config`) is pointing to the
 1. Create a "CF Installation Values" file and configure it:
    1. Use `./hack/generate-values.sh <system_domain>` to automatically generate values with [bosh interpolate](https://bosh.io/docs/cli-v2-install/#install)
    ```bash
-   $ ./hack/generate-values.sh cf.example.com > /tmp/cf-values.yml
+   $ ./hack/generate-values.sh <system_domain> > /tmp/cf-values.yml
```

Comment:
maybe if we tell the user to set `SYSTEM_DOMAIN=<system_domain>` at the start, then use that variable in all of the subsequent commands, reducing the copy/paste work.

Reply:
We're not sure the lifecycle of this script.  Let's not invest more than needed.  Also, we're using the angle brackets typography in this document.  Agreed that this means a little more typing from the end-user, right now.